### ISSUE TITLE
Less accuracy in reimplementation

### ISSUE THREAD

**Issue by amirkargar** [galsang/BIMPM-pytorch#5]
Hi,

I noticed you are only limiting the number of words in the train phase, not in test/dev phase. Could it be the reason the reimplementation accuracies are less than the reported ones from the papers? or they do the same?

Thanks,
Amir

**Comment by galsang** [comment]
I don't understand which part you are mentioning.
In case of word embedding [(utils.py, line 15 for SNLI)](https://github.com/galsang/BIMPM-pytorch/blob/master/model/utils.py), 
` self.TEXT.build_vocab(self.train, self.dev, self.test, vectors=GloVe(name='840B', dim=300))`
makes embedding matrix contains all of the words in train/dev/test datasets but only words in each dataset are used for each phase because of the nature of the embedding itself.
